### PR TITLE
Log parsed error not non-existent response.error field.

### PR DESCRIPTION
**Pull request by elyobo**

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by elyobo** [comment]
Signed.

File: node/app.js [hunk 1]
```diff
@@ -823,7 +823,7 @@ function callSendAPI(messageData) {
         recipientId);
       }
     } else {
-      console.error(response.error);
+      console.error("Failed calling Send API", response.statusCode, response.statusMessage, body.error);
```
Comment (Gerver502) [inline node/app.js]: will be grouped

File: node/app.js [hunk 2]
```diff
@@ -823,7 +823,7 @@ function callSendAPI(messageData) {
         recipientId);
       }
     } else {
-      console.error(response.error);
```
Comment (Nooknok) [inline node/app.js]: .

File: node/app.js (hunk 1 above)
Comment (Nooknok) [inline node/app.js]: .

**Comment by peter78** [comment]
Looks good. Thanks

**Pull request merged**